.github/workflows/alpine-release.yml

```diff
@@ -0,0 +1,34 @@
+name: Alpine Linux release
+on: 
+  push:
+    tags:
+      - v*
+env:
+  IMAGE_NAME: "opensmtpd"
+jobs:
+  build:
+    permissions: write-all
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v1
+    - name: Alpine Linux (amd64 musl openssl)
+      run: docker build . --file ci/docker/Dockerfile.alpine-release --tag opensmtpd:alpine
+
+    - name: Login to ghcr.io
+      run: echo "${{ secrets.GITHUB_TOKEN }}" | docker login ghcr.io -u $ --password-stdin
+
+    - name: Push Image
+      run: |
+        IMAGE_ID=ghcr.io/${{ github.repository_owner }}/$IMAGE_NAME
+        # Change all uppercase to lowercase
+        IMAGE_ID=$(echo $IMAGE_ID | tr '[A-Z]' '[a-z]')
+        # Strip git ref prefix from version
+        VERSION=$(echo "${{ github.ref }}" | sed -e 's,.*/\(.*\),\1,')
+        # Strip "v" prefix from tag name
+        [[ "${{ github.ref }}" == "refs/tags/"* ]] && VERSION=$(echo $VERSION | sed -e 's/^v//')
+        # Use Docker `latest` tag convention
+        echo IMAGE_ID=$IMAGE_ID
+        echo VERSION=$VERSION
```

Comment:
These two are just for debugging / providing a nicer output or forgot to `export` / continue the pipeline?